home02.py

```diff
@@ -0,0 +1,9 @@
+a = input("Write something: ")
+if (a.isdigit()) and (int(a) % 2 == 0):
+    print("It is number and it is even")
+elif (a.isdigit()) and (int(a) % 2 != 0):
+    print("It is number and it is odd")
+else:
+    print("It is text ")
```

Comment:
It would be much better to format this text in one line, like this: `print(f'It is a text. Its length is {len(a)}')`